packages/ra-ui-materialui/src/list/ListActions.tsx

```diff
@@ -115,7 +104,6 @@ ListActions.propTypes = {
 };
 
 export interface ListActionsProps extends ToolbarProps {
-    sort?: SortPayload;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**chore:** 'SortPayload' is imported but no more used‏, we may could remove it
(SortPayload was imported line 6 but I can't add a comment on)

Reply:
Right, fixed